packages/venia-ui/lib/components/CmsDynamicBlock/cmsDynamicBlock.ee.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { array, oneOf, oneOfType, string } from 'prop-types';
+import { gql, useQuery } from '@apollo/client';
+
+import ErrorView from '@magento/venia-ui/lib/components/ErrorView';
+
+import DynamicBlock from './dynamicBlock';
+
+export const DYNAMIC_BLOCK_FIXED_TYPE = `SPECIFIED`;
+export const DYNAMIC_BLOCK_SALES_RULE_TYPE = `CART_PRICE_RULE_RELATED`;
+export const DYNAMIC_BLOCK_CATALOG_RULE_TYPE = `CATALOG_PRICE_RULE_RELATED`;
+
+const getDynamicBlockType = displayMode => {
+    if (displayMode === 'fixed') {
```

Comment:
Maybe we could have constant variable for `fixed`, `salesrule `and `catalogrule`?